pandas/core/dtypes/dtypes.py

```diff
@@ -1542,6 +1543,25 @@ class BaseMaskedDtype(ExtensionDtype):
 
     base = None
     type: type
+    _internal_fill_value: Scalar
+
+    @property
+    def truthy_value(self):
```

Comment:
Should these be "private" ie `_truthy_value?

Reply:
they are accessed from outside the dtype's methods

Reply:
Shouldn't these raise NotImplementedError as a fallback? The idea is this class can be used for more than just float/integer/bool in the future right?

Reply:
i dont think there's any plans to extend like that

Reply:
On second thought decided to privatize after all